packages/ndla-ui/src/Resource/BlockResource.tsx

```diff
@@ -77,7 +78,40 @@ const ImageWrapper = styled.div`
   }
 `;
 
+const TagCounterWrapper = styled.p`
+  color: ${colors.brand.primary};
+  box-shadow: none;
+  margin: 0;
+  font-weight: 600;
+  ${fonts.sizes(16)}
+  &hover {
+    color: grey;
+  }
+`;
+
 const BlockResource = ({ link, title, tags, resourceImage, topics, description, actionMenu }: BlockResourceProps) => {
+  function CheckTagsLength(tags: string[]) {
+    if (tags.length > 3) {
+      return (
+        <>
+          <TagList tags={tags.slice(0, 3)} />{' '}
```

Comment:
Fjern mellomrom. Dersomd et er nødvendig for spacing, bruk gap/margin/padding heller.

Reply:
De kommer ofte automatisk. Tror det er prettifier hehe